xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -16,13 +16,24 @@ struct BacklinksView: View {
             HStack {
                 Text("Related Notes")
                     .font(.caption)
+                    .foregroundStyle(.secondary)
                 Spacer()
             }
             if backlinks.count > 0 {
-                TranscludeListView(
-                    entries: backlinks,
-                    onLink: onLink
-                )
+                LazyVStack(spacing: AppTheme.unit2) {
+                    ForEach(backlinks, id: \.self) { entry in
+                        VStack {
+                            TranscludeView(
+                                entry: entry,
+                                onLink: onLink
+                            )
+                            .buttonStyle(RelatedNoteButtonStyle(color: entry.color))
+                        }
+                        .tint(
+                            entry.headers.themeColor?.toHighlightColor()
+                        )
+                    }
+                }
```

Comment:
We need to apply different styles between transcludes in a note vs. the related notes section, so I forked off TranscludeListView and inlined it here.